modules/035-cni-simple-bridge/templates/daemonset.yaml

```diff
@@ -55,7 +55,7 @@ spec:
       serviceAccountName: {{ .Chart.Name }}
       containers:
       - name: simple-bridge
-        {{- include "helm_lib_module_container_security_context_read_only_root_filesystem_capabilities_drop_all_and_add" (list . (list "NET_ADMIN" "NET_RAW")) | nindent 8 }}
+        {{- include "helm_lib_module_container_security_context_privileged" . | nindent 8 }}
```

Comment:
Can we avoid running this container in the privileged mode?
I see that now both flannel and cilium use privileged mode, but I didn't get the cause.

Reply:
Sorry but no, because privileged mode is needed for create symlinks to proper version of iptables by iptables-wrapper.sh script on container start.